templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -549,11 +568,12 @@ contents:
       ovs-vsctl --timeout=30 --if-exists del-br br0
 
       # Make sure everything is activated
+      connections=()
       for connection in $(nmcli -g NAME c | grep -- "$MANAGED_NM_CONN_SUFFIX"); do
-        activate_nm_conn "$connection"
+        connections+=("$connection")
       done
-      activate_nm_conn ovs-if-phys0
-      activate_nm_conn ovs-if-br-ex
+      connections+=(ovs-if-phys0 ovs-if-br-ex)
```

Comment:
conflict: remove mentions of br-ex 1 and phys1 as 4.8 has no support for secondary bridge in configure-ovs.

Reply:
No `IFS=` whitespace fix?  I understand process reasons, but it doesn't seem to make sense to add code with known issues.  Or am I missing something in the backport?

Testing procedure is being updated to catch the old issues (whitespace in filenames, whitespace in ids, reboots after link failures).  It would be better to have more consistency in procedure than having to do testing tweaks.

Reply:
Ok, let's hold for that.

The reason I did not hold for it is that spaces in file names is a problem already present in 4.8 and I thought spaces in IDs are related and both happening when keyfiles are generated from ifcfg files.

Spaces in IDs by itself is still a problem on 4.8, but only for rollbacks.

Also note that there is people waiting for these fixes, we can't hold it forever. If problems are not regressions or not known to be affecting anybody, we might want to move forward.

Reply:
There is an ask to add whitespace tests in test automation, so I'm trying to figure out if the test deployment will fail constantly on 4.8 or not.